monai/bundle/config_item.py

```diff
@@ -262,34 +282,78 @@ def resolve_args(self):
         Utility function used in `instantiate()` to resolve the arguments from current config content.
 
         """
-        return {k: v for k, v in self.get_config().items() if k not in self.non_arg_keys}
+        return {
+            k: v
+            for k, v in self.get_config().items()
+            if (k not in self.non_arg_keys)
+            or (k == ConfigComponentReservedKeys.WRAPPER and not _wrapper_feature_flag.enabled)
```

Comment:
I don't think we should return the key and value under `k == ConfigComponentReservedKeys.WRAPPER and not _wrapper_feature_flag.enabled` this case, it will always wrong since the k will always not be the arguments for the instance?
Correct me if I misunderstand.

Reply:
This is meant for backwards compatibility. What if someone is already using the key _wrapper_ in one of their configs? SInce it was not reserved before, it might have been used in normal config. It's unlikely, but I wanted to make sure to cover this case. But I'm happy to always remove it. But in that case I would probably add a regex to check for "_\w+_" and make all keys of this pattern reserved for the future.